src/CalcViewModel/ApplicationViewModel.cpp

```diff
@@ -189,7 +189,7 @@ void ApplicationViewModel::OnPasteCommand(Object^ parameter)
     {
         ConverterViewModel->OnPasteCommand(parameter);
     }
-    else
+    else  if(NavCategory::IsCalculatorViewMode(m_mode))
```

Comment:
nit: spacing here is off

```suggestion
    else if (NavCategory::IsCalculatorViewMode(m_mode))
```